src/NuGet.Indexing/Downloads.cs

```diff
@@ -2,18 +2,37 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using System.Collections.Generic;
 using System.Linq;
 using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 using FrameworkLogger = Microsoft.Extensions.Logging.ILogger;
 
 namespace NuGet.Indexing
 {
-    public static class Downloads
+    public class Downloads
```

Comment:
the changes here are mostly to keep the code readable and sane. I moved away from IDictionary to a concrete class with just the needed method